CMakeLists.txt

```diff
@@ -13,6 +13,11 @@ option(ENABLE_PARAVIEW_CATALYST
 option(ENABLE_WARNINGS "Add -Wall to show compiler warnings" ON)
 option(ENABLE_EXTRA_WARNINGS "Add -Wextra to show even more compiler warnings" OFF)
 
+########################### NALU-first #####################################
+# Set Nalu's compilers, CMAKE_FIND_LIBRARY_PREFIXES
+# and CMAKE_FIND_LIBRARY_SUFFIXES, etc, after inheriting settings from Trilinos
```

Comment:
"after inheriting settings from Trilinos" is not really true anymore, correct?

Reply:
You mean that second line?

Reply:
What I means is that NaLU is no longer "inheriting settings from Trilinos" since it has to define the compilers **before** calling `find_package(Trilinos ...)`.